### DISCUSSION TITLE
Interested in developing applications for the platform, but I have some concerns

### DISCUSSION THREAD

**Discussion by MrPotatoBobx** [elementary/triage#125] [Q&A] [answered]
Just some simple questions, when Odin releases, will developers have an easy way to change their Debian packaging to flatpaks, and will developers have some kind of grace period before official release to do so? 

Do you think I should start learning how to make elementary applications and publish them now, or wait for Odins release?
 
Thanks in advance

**Comment by danirabbit** [comment, marked as answer]
Hey there! I'm not sure if there's any kind of tool to migrate deb packaging over to flatpak packaging available, but we do have a section in our developer guide already on writing a flatpak manifest here: https://docs.elementary.io/develop/writing-apps/our-first-app/packaging

There will definitely be time before the final release for developers to write their Flatpak manifests and submit their apps for publishing in AppCenter. The exact submission process is still being worked out currently, but we'll have more information soon about that on our blog.

One of the magical things about Flatpak is that you can start writing your app against the latest elementary platform already. If you're on elementary OS 5, you'll just need to add our repo here to get the SDK: https://flatpak.elementary.io/

I would recommend getting started sooner than later :)
